core/ProxyHttp.php

```diff
@@ -286,6 +286,10 @@ private static function compressFile($fileToCompress, $compressedFilePath, $comp
             $data = gzencode($data, 9);
         }
 
+        if (false === $data) {
```

Comment:
Not sure if we maybe should also check if the response looks valid, like checking if it has at least 50 chars or something like that 🤔

Reply:
If it's gzipped, that would be kind of hard, right?